pilot/pkg/networking/core/v1alpha3/route/route_test.go

```diff
@@ -1856,6 +1869,43 @@ var virtualServiceMatchingOnSourceNamespace = config.Config{
 	},
 }
 
+var virtualServiceWithStatPrefix = config.Config{
+	Meta: config.Meta{
+		GroupVersionKind: gvk.VirtualService,
+		Name:             "acme",
+	},
+	Spec: &networking.VirtualService{
+		Hosts: []string{},
+		Http: []*networking.HTTPRoute{
+			{
+				Name: "foo",
+				Match: []*networking.HTTPMatchRequest{
+					{
```

Comment:
Can you add another MatchRequest that does not have stat prefix configured and validate that route does not have it configured for completeness?

Reply:
@ramaraochavali done.